core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/Repeat.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.operation.impl;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.apache.commons.lang3.exception.CloneFailedException;
+
+import uk.gov.gchq.gaffer.operation.Operation;
+import uk.gov.gchq.gaffer.operation.Operations;
+import uk.gov.gchq.gaffer.operation.io.InputOutput;
+import uk.gov.gchq.gaffer.operation.serialisation.TypeReferenceImpl;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A {@code Repeat} is a Gaffer {@link Operation} for executing a delegate <code>Operation</code>
+ * for a specified number of repeats.
+ * <p>The delegate operation can also be an implementation of {@link Operations},
+ * so for example, a small {@link uk.gov.gchq.gaffer.operation.OperationChain} could be delegated
+ * to the <code>Repeat</code>, and repeated a number of times.</p>
+ */
+@JsonPropertyOrder(value = {"class", "input", "operation", "times"}, alphabetic = true)
+public class Repeat implements InputOutput<Object, Object>,
+        Operations<Operation> {
+    private Operation operation;
+    private int times;
```

Comment:
Perhaps create a constant - TIMES_DEFAULT, equal to 1. And then set 'times' to this default.